sonaranalyzer-dotnet/build/create-issue.ps1

```diff
@@ -100,7 +100,8 @@ function New-Issue() {
         -Uri $projectUri `
         -Method "POST" `
         -Headers $headers `
-        -Body ($payload | ConvertTo-Json)
+        -Body ($payload | ConvertTo-Json) `
+        -UseBasicParsing

```

Comment:
It allows the script to be used on machines where IE is not installed or where IE initial configuration is not performed.

Description from msdn:
"-UseBasicParsing Indicates that the cmdlet uses the response object for HTML content without Document Object Model (DOM) parsing."
We do not need to parse the DOM in scripts.